src/AsanaCurl.php

```diff
@@ -177,15 +179,15 @@ private function request($method, $url, array $parameters = [], array $headers =
 
         // Set options
         curl_setopt_array($curl, [
-            CURLOPT_URL => "{$this->endpoint}{$url}",
-            CURLOPT_CONNECTTIMEOUT => 10,
-            CURLOPT_TIMEOUT => 90,
-            CURLOPT_RETURNTRANSFER => 1,
-            CURLOPT_SSL_VERIFYPEER => 0,
-            CURLOPT_SSL_VERIFYHOST => 0,
-            CURLOPT_HEADER => 1,
-            CURLINFO_HEADER_OUT => 1,
-            CURLOPT_VERBOSE => 1,
+          CURLOPT_URL            => "{$this->endpoint}{$url}",
```

Comment:
Can you revert all of these array key/value alignments please, that way I can tell what really changed.

Reply:
What?  It 9 lines and nothing changes other than code formatting.

Reply:
Github clearly shows that no values changed, just spacing/tabbing around keys.

If I can't get this pull request merged, I'll move to making a package on packagist and call it done.  We've been having to operate off a dev release for months because of this.